proto/osmosis/concentrated-liquidity/pool-model/query.proto

```diff
@@ -117,6 +123,25 @@ message QueryLiquidityDepthsForRangeResponse {
   repeated LiquidityDepth liquidity_depths = 1 [ (gogoproto.nullable) = false ];
 }
 
+//=============================== TickLiquidityInBatches
+message QueryTickLiquidityInBatchesRequest {
+  uint64 pool_id = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  string lower_tick = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"lower_tick\"",
+    (gogoproto.nullable) = false
+  ];
+  string upper_tick = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"upper_tick\"",
+    (gogoproto.nullable) = false
+  ];
+  uint64 batch_unit = 4 [ (gogoproto.moretags) = "yaml:\"batch_unint\"" ];
```

Comment:
can you add a comment here explaining what batch unit means?

Also, your yaml is misspelled (unint instead of unit)

Reply:
https://github.com/osmosis-labs/osmosis/pull/4209/commits/14372a5abf09e4d22474e64c921c8b476bd3b282